web/modules/custom/rdf_entity/src/DataCollector/SparqlDataCollector.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+/**
+ * @file
+ * Integrate the Sparql query backend with the web profiler toolbar.
+ */
+
+namespace Drupal\rdf_entity\DataCollector;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Database\Database;
+use Drupal\rdf_entity\Database\Driver\sparql\Connection;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\webprofiler\DataCollector\DrupalDataCollectorTrait;
+use Drupal\webprofiler\DrupalDataCollectorInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+/**
+ * Class DatabaseDataCollector.
+ */
+class SparqlDataCollector extends DataCollector implements DrupalDataCollectorInterface {
```

Comment:
I didn't manage to get this code running, presumably because my detail page of my RDF entity is empty save from the title. I looked through the code but couldn't make much sense of it without context.